TheOrgBook/docker$ ./manage.sh build fails

hunk {0} polyfills.7a847f35bcd9a09c60c6.bundle.js (polyfills) 64.8 kB [initial] [rendered]
chunk {1} main.6ca46548b3a3265d4385.bundle.js (main) 104 kB [initial] [rendered]
chunk {2} styles.bd7c2783ce79f1c86e3d.bundle.css (styles) 1.61 kB [initial] [rendered]
chunk {3} vendor.ac1a3a3c9177edc85f3f.bundle.js (vendor) 583 kB [initial] [rendered]
chunk {4} inline.31e1fb380eb7cf3d75b1.bundle.js (inline) 798 bytes [entry] [rendered]

ERROR in /opt/app-root/src/src/app/roadmap/roadmap.component.ts (7,34): Module '"/opt/app-root/src/node_modules/compare-versions/index"' resolves to a non-module entity and cannot be imported using this construct.

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.1.2 (node_modules/watchpack/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.1.2 (node_modules/webpack-dev-server/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none was installed.
npm ERR! Linux 4.4.0-127-generic
npm ERR! argv "/opt/rh/rh-nodejs6/root/usr/bin/node" "/opt/rh/rh-nodejs6/root/usr/bin/npm" "install"
npm ERR! node v6.11.3
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! tob-web@0.0.1 postinstall: `ng build --prod --base-href ${NG_BASE_HREF:-/} ${NG_BUILD_FLAGS:-} && cp dist/themes/_active/index.html dist/index.html`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the tob-web@0.0.1 postinstall script 'ng build --prod --base-href ${NG_BASE_HREF:-/} ${NG_BUILD_FLAGS:-} && cp dist/themes/_active/index.html dist/index.html'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the tob-web package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ng build --prod --base-href ${NG_BASE_HREF:-/} ${NG_BUILD_FLAGS:-} && cp dist/themes/_active/index.html dist/index.html
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs tob-web
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls tob-web
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /opt/app-root/src/npm-debug.log
Build failed
ERROR: An error occurred: non-zero (13) exit code from centos/nodejs-6-centos7:6

--- a/.gitignore
+++ b/.gitignore
@@ -1,1 +1,4 @@
+# IntelliJ
+.idea
+
 *.pem

--- a/TheOrgBook/tob-web/package.json
+++ b/TheOrgBook/tob-web/package.json
@@ -27,7 +27,6 @@
     "@ngx-translate/core": "~8.0.0",
     "@ngx-translate/http-loader": "0.1.0",
     "bootstrap-sass": "~3.3.7",
-    "compare-versions": "^3.1.0",
     "core-js": "^2.4.1",
     "font-awesome": "^4.7.0",
     "localize-router": "git+https://github.com/cywolf/localize-router.git",

--- a/TheOrgBook/tob-web/src/app/roadmap/roadmap.component.ts
+++ b/TheOrgBook/tob-web/src/app/roadmap/roadmap.component.ts
@@ -4,7 +4,6 @@ import { TranslateService } from '@ngx-translate/core';
 import { GeneralDataService } from 'app/general-data.service';
 import { Location, LocationType, VerifiableOrg, VerifiableOrgType, IssuerService, VerifiableClaim, VerifiableClaimType, DoingBusinessAs,
   blankLocation, blankOrgType, blankLocationType, blankIssuerService, blankClaimType } from '../data-types';
-import * as compareVersions from 'compare-versions';
 
 @Component({
   selector: 'app-roadmap',
@@ -67,7 +66,6 @@ export class RoadmapComponent implements OnInit {
             let sname = regType.schemaName;
             let other = typesBySchema[sname];
 
-            // if(other && compareVersions(regType.schemaVersion, other.schemaVersion) <= 0)
             if(other && other.id < regType.id)
               continue;
 

